app/templates/admin/manageServiceLearningFaculty.html

```diff
@@ -111,37 +145,157 @@ <h5 class="ms-3">There are no approved courses for {{ term.description }}.</h5>
+                  </tr>
+                  <tr>
+                    <td class="fw-bold">Instructor(s)</td>
+                    <td><input id="courseInstructor" class="form-control" type="search" placeholder="Search instructor" value=""/></td>
+                    <!-- Start of Instructor Table -->
+                    <table class="table mb-3" id="instructorTable" name="instructorTable">
+                      <thead>
+                        <tr>
+                          <th scope="col">Instructor(s)</th>
+                          <th scope="col"></th>
+                        </tr>
+                      </thead>
+                      <tbody id="instructorTableBody">
+                          <tr data-username="" hidden>  <!-- This is a hidden table row. To add new rows to the instructor table, we copy this one and fill out the information -->
+                              <td>
+                                <p class="mb-0"></p>
+                                <input type="text" style="border: none" size="14" class="form-control-sm" id="" name="courseInstructorPhone" aria-label="Instructor Phone" data-value="" value="" placeholder="Phone Number" />
+                                <a class="text-decoration-none primary editButton" tabindex="0" data-username="" id=""  type="button">Edit</a>
+                              </td>
+                              <td class="align-middle">
+                                  <button id="remove" type="button" class="btn btn-danger removeButton">Remove</button>
+                              </td>
+                          </tr>
+  
+                          {% for instructor in courseInstructor %}
+                          <tr data-username="{{instructor.user.username}}">
+                            <td>
+                              <p class="mb-0">{{instructor.user.firstName}} {{instructor.user.lastName}} ({{instructor.user.email}})</p>
+                                <input type="text" style="border: none" size="14" class="form-control-sm" id="inputPhoneNumber-{{instructor.user}}" name="courseInstructorPhone" aria-label="Instructor Phone" data-value="{{instructor.user.phoneNumber}}" value="{{instructor.user.phoneNumber}}" placeholder="Phone Number" />
+                                <a class="text-decoration-none primary editButton" tabindex="0" data-username="{{instructor.user.username}}" id="editButton-{{instructor.user}}" type="button">Edit</a>
+                            </td>
+                            <td class="align-middle">
+                                <button id="remove" type="button" class="btn btn-danger removeButton">Remove</button>
+                            </td>
+                          </tr>
+                          {% endfor %}
+                      </tbody>
+                    </table>
+                    <!-- End of Instructor Table -->
+                  </tr>
```

Comment:
This doesn't close on the same indentation level as it opens on line 242. Do you think you could fix that?

Reply:
Fixed